server/routes/createGoal/createGoalController.test.ts

```diff
@@ -220,17 +255,57 @@ describe('createGoalController', () => {
     })
   })
 
+  describe('submitAddNoteForm', () => {
```

Comment:
This test was missing, so I've added it (because I had to make a change to the implementation in the controller, so made sense to test it)